manifests/q/Qingfeng/HeyboxAccelerator/1.1.19/Qingfeng.HeyboxAccelerator.locale.en-US.yaml

```diff
@@ -0,0 +1,38 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Qingfeng.HeyboxAccelerator
+PackageVersion: 1.1.19
+PackageLocale: en-US
+Publisher: Qingfeng
+PublisherUrl: https://open.heybox.hk/
+PublisherSupportUrl: https://acc.xiaoheihe.cn/help/center
+PrivacyUrl: https://api.xiaoheihe.cn/account/acc_privacy_introduce/?type=2
+Author: Qingfeng (Beijing) Technology Co., Ltd.
+PackageName: 小黑盒加速器
```

Comment:
`PackageName: Heybox Accelerator`

Reply:
`PackageName` corresponds to the `DisplayName` of the ARP entry the installer wrote to the registry.